code/modules/multiz/movement.dm

```diff
@@ -12,6 +12,28 @@
 	if(zMove(DOWN))
 		to_chat(src, "<span class='notice'>You move down.</span>")
 
+/mob/verb/lookup()
+	set name = "Look Up"
+	set category = "IC"
+
+	if(HasAbove(src.z))
+		var/turf/T = GetAbove(src)
+		if(client.eye == T)
+			lookup_reset_view()
+		else if(T && isvisiblespace(T))
+			reset_view(GetAbove(src))
+			to_chat(src, "<span class='notice'>You look upwards.</span>")
+			register_event(/event/moved, src, nameof(src::lookup_reset_view()))
+		else
+			to_chat(src, "<span class='warning'>You see nothing but the ceiling.</span>")
+	else
+		to_chat(src, "<span class='warning'>You don't feel like doing that.</span>")
```

Comment:
I'd rather this verb just be disabled on non-multiz maps if possible

Reply:
done